### PR TITLE
Improve bucket creation test

### PR DESCRIPTION
**Pull request by MeridianExplorer**
Improve the test by actually verifying that the created buckets actually have a healthy status and aren't just created.

### CODE REVIEW

File: tests/manage/mcg/helpers.py [hunk 1]
```diff
@@ -352,3 +355,23 @@ def s3_delete_object(s3_obj, bucketname, object_key):
 
     """
     return s3_obj.s3_client.delete_object(Bucket=bucketname, Key=object_key)
+
+
+def wait_for_mcg_resource_status(resource_name, resource_type, column, status, interval, timeout):
```
[the remark this answers is not in the document]
Comment (MeridianExplorer) [reply tests/manage/mcg/helpers.py]: The function was omitted in the final version, and thus no docstring was added.
Thanks for the review!

**Comment by MeridianExplorer** [comment]
@fbalak 
I'll appreciate if you can review this when possible.
I changed your `test_noobaa.py` and replaced all `bucket.phase` calls with `bucket.status`, and removed the `phase` property.
The reason is that S3 buckets don't have a phase, and don't appear in the Kubernetes/OC control plane at all, which might lead to unexpected behavior in the future, or when used by others who don't know this. 
I simply took your `phase` code and put it inside the `OCBucket` `status` function.

I also ran `test_noobaa.py` locally on my AWS cluster, and it passed successfully after the changes - 

> =============================== warnings summary ===============================
myvenv/lib/python3.7/site-packages/_pytest/config/__init__.py:891
  /home/newuser/repos/ocs-ci-personal-fork/myvenv/lib/python3.7/site-packages/_pytest/config/__init__.py:891: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: pytest_reportportal
    self._mark_plugins_for_rewrite(hook)
-- Docs: https://docs.pytest.org/en/latest/warnings.html
=================== 2 passed, 1 warning in 944.71s (0:15:44) ===================

File: ocs_ci/ocs/constants.py [hunk 2]
```diff
@@ -74,6 +74,10 @@
 # NooBaa statuses
 BS_AUTH_FAILED = 'AUTH_FAILED'
 BS_OPTIMAL = 'OPTIMAL'
+HEALTHY_OB = 'OPTIMAL'
+HEALTHY_OBC = STATUS_BOUND
+HEALTHY_OBC_CLI_PHASE = 'Phase                  : Bound'
```
Comment (fbalak) [inline ocs_ci/ocs/constants.py:79]: Is there really so many whitespaces between `Phase` and `:`? Isn't it a bug? Maybe it should be:
```
`'Phase:                   Bound'`
```

**Comment by MeridianExplorer** [comment]
Blocked by #1828 since it relies on the new `exec_mcg_cmd` command

File: ocs_ci/ocs/constants.py (hunk 2 above)
Comment (MeridianExplorer) [reply ocs_ci/ocs/constants.py:79]: Not a bug - it's used to prettify the CLI prints - 
```
Bucket status:
  Name                   : oc-bucket-1b267097d1ed4f40aab0eb041b5fdde1
  Type                   : REGULAR
  Mode                   : OPTIMAL
```

```
ObjectBucketClaim info:
  Phase                  : Bound
```

Comment (MeridianExplorer) [reply ocs_ci/ocs/constants.py:79]: I tried to come up with a way to parse the CLI output, but it's simply not machine readable enough. Thus, I chose to go with space stripping.

**Comment by MeridianExplorer** [comment]
PR job is acting weird, verified the run locally - 
```
=============================== warnings summary ===============================
myvenv/lib/python3.7/site-packages/_pytest/config/__init__.py:891
  /home/newuser/repos/ocs-ci-personal-fork/myvenv/lib/python3.7/site-packages/_pytest/config/__init__.py:891: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: pytest_reportportal
    self._mark_plugins_for_rewrite(hook)

tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[100-S3]
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[1000-S3]
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[100-OC]
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[1000-OC]
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[100-CLI]
tests/manage/mcg/test_bucket_creation.py::TestBucketCreation::test_bucket_creation[1000-CLI]
  /home/newuser/repos/ocs-ci-personal-fork/myvenv/lib/python3.7/site-packages/pytest_reportportal/listener.py:50: DeprecationWarning: cgi.escape is deprecated, use html.escape instead
    cgi.escape(report.longreprtext),

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============= 6 passed, 6 skipped, 7 warnings in 364.58s (0:06:04) =============
```

File: ocs_ci/ocs/constants.py (hunk 2 above)
Comment (fbalak) [reply ocs_ci/ocs/constants.py]: Ok

**Pull request merged**